pkg/collector/corechecks/net/ntp.go

```diff
@@ -42,7 +42,7 @@ type ntpInstanceConfig struct {
 	OffsetThreshold       int      `yaml:"offset_threshold"`
 	Host                  string   `yaml:"host"`
 	Hosts                 []string `yaml:"hosts"`
-	Port                  string   `yaml:"port"`
+	Port                  int      `yaml:"port"`
```

Comment:
If `port` is now an integer, we should update the `ntp` check configuration [here](https://github.com/DataDog/datadog-agent/blob/51d52df2774ced9b141d70b9dc4bc60361976612/cmd/agent/dist/conf.d/ntp.d/conf.yaml.default#L17) to not mislead our users into thinking they can put strings here.

Reply:
Good catch!

Reply:
I think you will also need to update https://github.com/DataDog/integrations-core/blob/master/ntp/datadog_checks/ntp/data/conf.yaml.default. The situation is pretty confusing for this check as we had a python version for agent5 which is not used for agent6 so we end up having two places for the configuration.
I don't know from where the config shipped in the final package is.

Reply:
Didn't you remove this file in your PR?